utils/singlejs/.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
```

Comment:
Do we need this? .DS_Store and build are already in the parent .gitignore.

Reply:
It would be great to have the `.gitignore` per folder. For example, if we change the output folder from `build` to `dist`, we can modify only the `.gitignore` in the project.